Documentation/demo.md

```diff
@@ -271,7 +271,10 @@ etcdctl --endpoints=$ENDPOINTS endpoint health
 
 <img src="https://storage.googleapis.com/etcd/demo/11_etcdctl_snapshot_2016051001.gif" alt="11_etcdctl_snapshot_2016051001"/>
 
+Snapshot can only be requested from one etcd node, so `--snapshot` flag should contain only one endpoint.
```

Comment:
@vimalk78 sorry I missed it earlier as this reads nicely, but there is no `--snapshot` flag so instead of it we should say `snapshot` command or `--endpoints` flag ? Home makes sense. Thanks!

Reply:
thats why there are reviews.
Thanks for catching.